src/Actions/searchResultActions.js

```diff
@@ -1,5 +1,10 @@
 import * as types from './types';
 
+export const searchResultPageLoad = params => ({
```

Comment:
Just to Question : If possible, can we pass `params` to `searchResultRequest` action, that way we dont have create this action. It seems they are doing the same.

Reply:
Yeah I remember discussing this with him as well... now I am seeing how these are redundant. What do you think? @Jake-Lindvall

Reply:
Did you mean pass it to `searchHandleInput` action? As it is right now, the logic in `SearchResults` looks like:

```
if (query) {
  searchResultPageLoad(query);
  searchResultRequest();
}
```

For a little clarity, here's why I have it as a separate action, though:
* `searchHandleInput` seems to depend on `e.target.value`, and I didn't want to refactor everything around that
* `searchResultPageLoad` makes it simpler and runs every page load, looking to update the state if possible and avoids being concerned at all with pretending there was a user input
* Having a separate action and action type is clearer for debugging and other semantic purposes - it isn't an input, it really is just a page 

If that still seems silly, I totally understand and can update it 👍

Reply:
I still did not see the drop-down for users when you go to the Request Assets part of the App. Other than that all the logic and routing look great to me. There might be a little refactoring due to how we now handle the search icon now but once that gets revised I will approve.

Reply:
@robertwollny-pnmac it's due to the merge conflicts on his remote branch. Just confirmed with @Jake-Lindvall that the dropdown works on his local branch with the merge conflicts resolved 👍

Reply:
Pushing up changes; test it out and approve if it looks good

@robertwollny-pnmac Agreed on the refactoring - things behave slightly different now, because clicking the menu item for search directs right to `/search`, but it shouldn't be too much future work :)